test/nodeMock/ccd/judicialBookings_data.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
```

Comment:
Looking in the folder most of the files don't have _data prefixed on them, so for consistency can you remove _data from the filename.

Reply:
Ideally call it judicialBooking.js (singular)

Reply:
this file shouldn't exist, removed .